### DISCUSSION TITLE
Import Scrtipt does not work

### DISCUSSION THREAD

**Discussion by thgeese** [tteck/Proxmox#1049] [Help Needed]
**Describe the issue**
Trying to import a backup from other installation. Got this error.

Traceback (most recent call last):
  File "/opt/paperless/src/documents/management/commands/document_importer.py", line 17, in <module>
    from documents.models import Document
ModuleNotFoundError: No module named 'documents'

**Comment by tteck** [comment]
Let me see if I've got this right: you have the paperless-ngx software running inside an LXC container and now you want to import a backup into paperless-ngx. Is that correct?

**Reply by thgeese** [reply]
> Let me see if I've got this right: you have the paperless-ngx software running inside an LXC container and now you want to import a backup into paperless-ngx. Is that correct?

correct

**Comment by tteck** [comment]
Maybe `pip install documents` ?

**Comment by thgeese** [comment]
yes tried that but that did not fix the issue. I saw that the rights are set to user 1001 which does not exist.

**Reply by steve1123456** [reply]
> yes tried that but that did not fix the issue. I saw that the rights are set to user 1001 which does not exist.

Hi there. Wondering if you ever got this to work? I'm having a similar issue; trying to use document_importer to import a backup that was done on my old paperless-ngx installation using document_exporter.

Thanks.
